packages/plan-form/src/Form/index.tsx

```diff
@@ -158,6 +159,24 @@ export interface PlanFormProps extends CommonProps {
   ) => string /** callback to get the path to redirect after successfully form submission */;
 }
 
+/**
+ * Plan Form component
+ *
```

Comment:
This description does not match what the function does. Also, could we add more details to the description? e.g what dates are disabled?